### DISCUSSION TITLE
Role model expansion

### DISCUSSION THREAD

**Discussion by yiannisccmath** [Permify/permify-gorm#4] [Q&A]
Hello, it's a nice library and I'm thinking of using it to my project. I only have 2 questions

1. Why isn't there an UpdateRole method only for updating name and description of a role? Is this forbidden to update them for some reason?

2. Can I somehow expand the role model to include CreatedBy and ModifiedBy properties that will be updated with the ID of my user model?

Thanks in advance!

**Comment by tolgaozen** [comment]
Hello Yiannis,

First of all, thank you for your interest.

For the 1st question, we didn't add it because we think that name changing it might cause a problem because you can fetch role with name field. In addition, we didn't think that it was a situation that would be needed much. But we can add the description update as soon as possible.

For question 2, we did not add any expandable options for role model, but we can add this idea to the road map. I think it could be a useful feature.

**Comment by yiannisccmath** [comment]
Thank you for the quick reply! Is the road map public so we can see what's coming and the date of the feature addition?

**Comment by tolgaozen** [comment]
We are preparing a web page about road map, we will share it as soon as possible.
